custom-targets/infrastructure-manager/README.md

```diff
@@ -0,0 +1,75 @@
+# Cloud Deploy Infrastructure Manager Terraform Deployer Sample
+This directory contains a sample implementation of a Cloud Deploy Custom Target for deploying Google
```

Comment:
Add documentation about use case, for example:

"This enables you to use Cloud Deploy to control the application of Infrastructure Manager via a Cloud Deploy Delivery Pipeline. Some example use cases:

* Manage your infrastructure changes form the same interface as the rest of your application delivery
* Control the progression of infrastructure changes between dev, test and production environments (manually or via automation)
* Run tests to validate that changes are applied successfully
* Use any any other Cloud Deploy features alongside Deployment Manager (i.e. pre/post deployment tasks, approvals, etc...)

Reply:
Updated